DelfinPerformance/README.md

```diff
@@ -0,0 +1,67 @@
+## Delfin Performance Monitoring with Preometheus
+
+This is demo of SODA Foundations Delfin performance monitoring feature using default preometheus exporter.
+
+
+Delfin is a heterogeneous infrastructure management platform. It manages and monitors multiple storage backends from a single place. Prometheus is default integrated with delfin for monitoring performances of the storages.
+
+The detailed analysis and architecture of prometheus integration with delfin can be found [here](https://github.com/sodafoundation/design-specs/blob/dcdee7b67d4a4ee74f065f00b2e93efb22f2493a/specs/SIM/PerfomanceMontoringDesign.md)
```

Comment:
What is relationship of the design document and this example? Please detail it. Otherwise, user will be confused

Reply:
done